src/sidebar/services/test/groups-test.js

```diff
@@ -209,6 +231,10 @@ describe('groups', function() {
           fakeApi.groups.list,
           sinon.match({ expand: ['organization', 'scopes'] })
         );
+        assert.calledWith(
+          fakeApi.group.read,
+          sinon.match({ expand: ['organization', 'scopes'] })
```

Comment:
It doesn't look like there is a test in this file to check that the right group ID is passed to `api.group.read`.